deploy/dcos/PulsarGroups.json

```diff
@@ -0,0 +1,351 @@
+          "cpus": 1,
+          "mem": 1024,
+          "instances": 3,
+          "constraints": [["hostname", "UNIQUE"]],
+          "container": {
+            "type": "DOCKER",
+
+            "docker": {
+              "image": "apachepulsar/pulsar:latest",
+              "network": "HOST"
+            }
+          },
+
+          "portDefinitions": [
+            {
+              "protocol": "tcp",
+              "port": 6650
+            },
+            {
+              "protocol": "tcp",
+              "port": 8082,
+              "labels": {
+                "METRICS_PATH": "/metrics"
+              }
+            }
+          ],
+
+          "requirePorts": true,
+
+          "env": {
+            "// Notice": "add PULSAR_MEM, PULSAR_GC, according to your environment.",
+            "webServicePort": "8082",
+            "zookeeperServers":  "master.mesos:2181",
+            "globalZookeeperServers":  "master.mesos:2181",
+            "clusterName": "pulsar-dcos"
+          },
+
+          "args": ["sh", "-c", "python bin/apply-config-from-env.py conf/broker.conf && python bin/apply-config-from-env.py conf/pulsar_env.sh && bin/pulsar broker"],
+
+          "healthChecks": [
+            {
+              "protocol": "TCP",
+              "portIndex": 0,
+              "gracePeriodSeconds": 300,
+              "intervalSeconds": 60,
+              "timeoutSeconds": 20,
+              "maxConsecutiveFailures": 0,
+              "ignoreHttp1xx": false
+            }
+          ],
+
+          "upgradeStrategy": {
+            "maximumOverCapacity": 0,
+            "minimumHealthCapacity": 0
+          }
+        },
+        {
+          "id": "/pulsar/brokers/brokeradmin",
+          "cpus": 1,
+          "mem": 1024,
+          "instances": 1,
+          "constraints": [["hostname", "UNIQUE"]],
+          "container": {
+            "type": "DOCKER",
+
+            "docker": {
+              "image": "apachepulsar/pulsar:latest",
+              "network": "HOST"
+            }
+          },
+
+          "env": {
+            "webServicePort": "8082",
+            "zookeeperServers":  "master.mesos:2181",
+            "globalZookeeperServers":  "master.mesos:2181",
+            "clusterName": "pulsar-dcos",
+            "webServiceUrl": "http://broker.brokers.pulsar.marathon.mesos:8082",
+            "brokerServiceUrl":  "pulsar://broker.brokers.pulsar.marathon.mesos:6650"
+          },
+
+          "args": ["sh", "-c", "bin/apply-config-from-env.py conf/client.conf && sleep 10000000000"]
+        }
+      ]
+    },
+
+    {
+      "id": "/pulsar/monitor",
+      "dependencies": ["/pulsar/brokers/broker"],
+      "apps": [
+        {
+          "id": "/pulsar/monitor/prom",
+          "cpus": 0.5,
+          "mem": 512,
+          "instances": 1,
+          "constraints": [["hostname", "UNIQUE"]],
+          "container": {
+            "type": "DOCKER",
+
+            "docker": {
+              "image": "zhaijia/prometheus",
```

Comment:
is there any thing additional present into docker image compare to `https://hub.docker.com/r/prom/prometheus/` ? If not then should we replace it with `prom/prometheus/`?

Reply:
Yes, the configuration file is special for DC/OS

Reply:
then should we we add docker file in this repo and push image under `apachepulsar`?

Reply:
+1 to @rdhabalia suggested.